src/azure-cli/azure/cli/command_modules/appservice/_create_util.py

```diff
@@ -337,6 +337,7 @@ def get_site_availability(cmd, name):
 def get_app_details(cmd, name):
     client = web_client_factory(cmd.cli_ctx)
     data = (list(filter(lambda x: name.lower() == x.name.lower(), client.web_apps.list())))
+    print(data)
```

Comment:
If it is debugging purpose, use logger. If it is command output, consider returning data in a method. Azure CLI will print them. We seldom print data by self.